__tests__/auth.test.ts

```diff
@@ -28,7 +28,7 @@ describe('Given an event with an [INCORRECT] scope', () => {
 
     const authInstance = new Auth();
 
-    expect(authInstance.checkScopesAndResolve(mockedEvent, ['incorrect scope'])).resolves.toThrow('Error: You are not authorized!');
+    await expect(authInstance.checkScopesAndResolve(mockedEvent, ['incorrect scope'])).resolves.toThrow('Error: You are not authorized!');
```

Comment:
This makes sure that the code here actually waits for the promise to resolve before going to the next line and stopping the jwksMock.